### PR TITLE
[stable/home-assistant] adding runningman84 to OWNERS

### PR DESCRIPTION
**Pull request by billimek**
#### What this PR does / why we need it:

Adding @runningman84 as a chart maintainer

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

**Comment by billimek** [comment]
Helping resolve #13867

File: stable/home-assistant/OWNERS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 approvers:
 - billimek
+- runningman84
 reviewers:
 - billimek
+- runningman84
```
Comment (maorfr) [inline stable/home-assistant/OWNERS]: missing a newline here

Comment (billimek) [reply stable/home-assistant/OWNERS]: done!

**Comment by maorfr** [comment]
/lgtm

thanks and good luck!

**Comment by runningman84** [comment]
Thanks

**Pull request merged**